src/adr/0036-eventbus.md

```diff
@@ -0,0 +1,73 @@
+# 36. Event bus
+
+Date: 2020-05-05
+
+## Status
+
+Accepted (lazy consensus)
+
+## Context
+
+James mailbox uses an event driven architecture. 
```

Comment:
what is James mailbox?

Reply:
James mailbox core components.

The definition of the mailbox core components, entities, responsibilities, sub-components and implementations might be a really good topic for yet-another ADR but I must admit I reach too much nested ADR to be happy to write yet-another one.

What explanation/dis-ambiguation do you expect?

Reply:
James mailbox is a technical module in our code. I find it a little harsh in a "Context" part of an ADR.
I thus suggest that you use a more generic term, of that you remove "mailbox" from this first sentence.

Reply:
Let me disagree.

The eventbus so far is a mailbox sub-component. The proof is that its interface is in mailbox-api. So I cannot speak of the eventbus without this context.

Generifying it to unbound it from aits mailbox context may have some benefits but it is a decision still to make (and will require an ADR).

So then I see no other solution than to write a "define me the mailbox ADR"...

Reply:
I'm not sure you really need an ADR just to define one term. My fear is that without knowing that "mailbox" is a core James component, one can understand this sentence with mailbox meaning user mailbox, and hence it makes no sense. Cannot we just clarify this mailbox thing?

Maybe just `James mailbox component` would do the trick, or to be better selft explaining `James mailbox component, a core component of James handling the storage of mails and mailboxes,`...